app/routes/header.tsx

```diff
@@ -29,27 +29,34 @@ export function Header({
               <img alt="bball" src="/assets/basketball-solid.svg" />
             </div>
           </div>
-          <Link to="/faq" className="h2" style={{ paddingTop: 10 }}>
-            learn more
-          </Link>
         </div>
         <div className="header_right">
           {hideLoginLinks ? null : user ? (
             <>
               <p className="header_user">{user.email}</p>
-              <Form action="/logout" method="post">
-                <button
-                  type="submit"
-                  className="header_link header_link___button"
-                >
-                  Sign out
-                </button>
-              </Form>
+              <div className="header_rightGroup">
+                <Form action="/logout" method="post">
+                  <button
+                    type="submit"
+                    className="header_link header_link___button"
+                  >
+                    Sign out
+                  </button>
+                </Form>
+                <Link className="header_link___learnMore" to="/faq">
+                  Learn more
+                </Link>
+              </div>
             </>
           ) : (
-            <Link to="/start" className="header_link header_link___button">
-              Sign up or log in
-            </Link>
+            <div className="header_rightGroup">
+              <Link to="/start" className="header_link header_link___button">
+                Sign up or log in
+              </Link>
+              <Link className="header_link___learnMore" to="/faq">
+                Learn more
+              </Link>
```

Comment:
i'm not worried about the repetition, but the paradigmatic way to make it feel a little dry-er would be to just encapsulate it in its own little component, so that if you ever need to change the HTML a bit, you'll only have to do it once.

```tsx
const LearnMore = () => (
  <Link className="header_link___learnMore" to="/faq">
    Learn more
  </Link>
)

// ...
<LearnMore />
```